### PR TITLE
Update about-github-packages.md

### PR DESCRIPTION
**Pull request by verhas**
### Why:

The sentence makes no sense.

### What's being changed:

One sentence in the documentation.

### Check off the following:
- [x ] All of the tests are passing. this is a documentation change
- [x ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@verhas Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Comment by janiceilene** [comment]
Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions ✨

**Comment by verhas** [comment]
What is the reason to ask for PRs if you both do not attend to it and the same time have automation that closes the request?

Should I bother to suggest fix for anything when a simple thing, like this, fixing a sentence in a documentation gets stale?

**Comment by janiceilene** [comment]
Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions and got quite backed up between the US Thanksgiving holiday, GitHub Universe, and our team taking some time off during the December holidays.

This PR has been triaged to the appropriate team and they'll take a look as soon as they can.

**Comment by DozerKatDC10** [comment]
> Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions and got quite backed up between the US Thanksgiving holiday, GitHub Universe, and our team taking some time off during the December holidays.
> 
> This PR has been triaged to the appropriate team and they'll take a look as soon as they can.

Thanks Janice took care of signing in as you can see. Thank you again for you and the whole Team's wonderfulo effert's.

File: content/packages/learn-github-packages/about-github-packages.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ For more information, see:
 
 ### Managing packages
 
-You can delete a version of a private package on {% data variables.product.product_name %} or using the GraphQL API. When you use the GraphQL API to query and delete private packages, you must use the same token you use to authenticate to {% data variables.product.prodname_registry %}. For more information, see "[Deleting a package](/packages/manage-packages/deleting-a-package)" and "[Forming calls with GraphQL](/graphql/guides/forming-calls-with-graphql)."
+You can delete a version of a private package on {% data variables.product.product_name %} via the user interface or using the GraphQL API. When you use the GraphQL API to query and delete private packages, you must use the same token you use to authenticate to {% data variables.product.prodname_registry %}. For more information, see "[Deleting a package](/packages/manage-packages/deleting-a-package)" and "[Forming calls with GraphQL](/graphql/guides/forming-calls-with-graphql)."
```
Comment (runleonarun) [inline content/packages/learn-github-packages/about-github-packages.md:148]:
```suggestion
You can delete a version of a private package in the {% data variables.product.product_name %} user interface or using the GraphQL API. When you use the GraphQL API to query and delete private packages, you must use the same token you use to authenticate to {% data variables.product.prodname_registry %}. For more information, see "[Deleting a package](/packages/manage-packages/deleting-a-package)" and "[Forming calls with GraphQL](/graphql/guides/forming-calls-with-graphql)."
```

**Review by runleonarun — APPROVED** [review]
@verhas Thanks for your fix and your patience!

**Pull request merged**